### ISSUE TITLE
Wiki- inconsistent descriptions hard to follow

### ISSUE THREAD

**Issue by ashreves** [episphere/quest#57]
Throughout the wiki, some of the descriptions were very detailed and clear and others were hard to follow. I recommend descriptions be kept more consistent to make the wiki easier to follow.

Examples of two very different styles-
Syntax clearly described:
|__|__| - Pipe, underscore, underscore, pipe, underscore, underscore, pipe indicates a numeric response.

Syntax implied:
[textbox: VARIABLE_NAME] – Added at the end of a response option will display a follow-up text box if that response option is selected

**Comment by hzhao392** [comment]
@ruggz13 take a look at wiki and verify the descriptions are clear
